README.md

```diff
@@ -26,9 +26,10 @@ If you want to switch back to sunxi versions for some reasons (no device tree av
 - change the file conf/machine/include/sunxi.inc to include the following block
 - edit your conf/local.conf to add the following block
 
-	PREFERRED_PROVIDER_u-boot="u-boot-sunxi"
-	PREFERRED_PROVIDER_virtual/bootloader="u-boot-sunxi"
-	PREFERRED_PROVIDER_virtual/kernel="linux-sunxi"
+		PREFERRED_PROVIDER_u-boot="u-boot-sunxi"
```

Comment:
can you please put alignment as previous version? thanks

Reply:
I fixed alignment to make it render properly in github. Currently this part is unformatted:

![image](https://user-images.githubusercontent.com/59499/73132777-388f3400-4028-11ea-8114-fc02d8de9943.png)

That said, do you still want me to restore previous indentation?

Reply:
Sorry I overlooked it. No keep it like you have it in PR.